ui/page/exchange/create_order_page.go

```diff
@@ -128,6 +130,8 @@ func NewCreateOrderPage(l *load.Load) *CreateOrderPage {
 			Alignment: layout.Middle,
 			Axis:      layout.Vertical,
 		}},
+		startTradingBtn: l.Theme.Button(values.String(values.StrStartTrading)),
+		isFirstVisit:    true,
```

Comment:
Hmm, so always true until you click start trading?

Reply:
Yes. We don't; save to asset manager db atm.